lib/webhooks/register.ts

```diff
@@ -149,10 +150,17 @@ async function getExistingHandlers(
   return existingHandlers;
 }
 
-function buildCheckQuery(endCursor: string | null) {
-  return queryTemplate(TEMPLATE_GET_HANDLERS, {
+function buildCheckQuery(config: ConfigInterface, endCursor: string | null) {
+  let query = queryTemplate(TEMPLATE_GET_HANDLERS, {
     END_CURSOR: JSON.stringify(endCursor),
   });
+
+  // The privateMetafieldNamespaces field was deprecated in the July22 version, so we need to stop sending it
```

Comment:
just a style suggestion, but if we created a private function called `removeDepricatedFields(query)` you give yourself an explanation as to why you are modifying the query as well as room to grow as more fields get invariably depreciated in the future. The a well named function like that the, your well meaning comment is moot as your intent is easy to follow by reading the code.